easybuild/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-2.1.2-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,104 @@
+    ('double-conversion', '3.3.0'),  # for torchtext
+    ('utf8proc', '2.8.0'),  # for torchtext
+    ('tensorboard', '2.15.1'),  # for torch-tb-profiler
+    ('FFmpeg', '6.0'),  # for torchvision and torchaudio
+    ('SoX', '14.4.2'),  # for torchaudio
+]
+
+use_pip = True
+
+exts_list = [
+    ('portalocker', '2.8.2', {
+        'checksums': ['2b035aa7828e46c58e9b31390ee1f169b98e1066ab10b9a6a861fe7e25ee4f33'],
+    }),
+    ('torchdata', '0.7.1', {
+        'preinstallopts': "USE_SYSTEM_LIBS=1 ",
+        'source_urls': ['https://github.com/pytorch/data/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['ef9bbdcee759b53c3c9d99e76eb0a66da33d36bfb7f859a25a9b5e737a51fa23'],
+        'runtest': False,  # circular test requirements
+    }),
+    ('torchtext', '0.16.2', {
+        'patches': ['torchtext-0.14.1_use-system-libs.patch'],
+        'source_urls': ['https://github.com/pytorch/text/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': [
+            {'torchtext-0.16.2.tar.gz': '6574b012804f65220329a2ad34a95c18e4df0d4cff2f862fb7862d57b374b013'},
+            {'torchtext-0.14.1_use-system-libs.patch':
+             '1366d10c4755b6003194f7313ca11d165a80a13d325bee9d669ea2b333d82536'},
+        ],
+        'runtest': 'pytest',
+        'testinstall': True,
+    }),
+    ('torchvision', '0.16.2', {
+        'preinstallopts': 'WITH_CUDA=1',
+        'installopts': '-v',
+        'patches': ['torchvision-0.16.2_ffmpeg-6.0-fix.patch'],
+        'source_urls': ['https://github.com/pytorch/vision/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': [
+            {'torchvision-0.16.2.tar.gz': '8c1f2951e98d8ada6e5a468f179af4be9f56d2ebc3ab057af873da61669806d7'},
+            {'torchvision-0.16.2_ffmpeg-6.0-fix.patch':
+             'a49336e7bfa1c950e886852bff37a3ea2146ac7bda87241e3ffb31c5cb869cce'},
+        ],
+        'runtest': 'pytest',
+        'testinstall': True,
+    }),
+    ('torchaudio', '2.2.1', {
+        'installopts': '--no-use-pep517 -v',
+        'patches': ['torchaudio-2.2.1_use-external-sox.patch'],
+        'preinstallopts': ('rm -r third_party/{sox,ffmpeg/multi} &&'
+                           ' USE_CUDA=1 USE_OPENMP=1 USE_FFMPEG=1 FFMPEG_ROOT="$EBROOTFFMPEG"'),
+        'source_urls': ['https://github.com/pytorch/audio/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': [
+            {'torchaudio-2.2.1.tar.gz': '5f2a9cda9369ff348d1d7ca2c0a1f01afc9ebff1a13df9775c22aa5c43f5b0f1'},
+            {'torchaudio-2.2.1_use-external-sox.patch':
+             'fc97b2bdf0ab16b9c4706bf2a31dd72d0ecb09e4af2f9c4a34e18fca8e0d165d'},
+        ],
+        'runtest': 'pytest',
```

Comment:
```suggestion
        'runtest': (
            'pytest'  # ignored tests require librosa
            ' --ignore=test/torchaudio_unittest/functional/librosa_compatibility_cpu_test.py'
            ' --ignore=test/torchaudio_unittest/functional/librosa_compatibility_cuda_test.py'
            ' --ignore=test/torchaudio_unittest/prototype/functional/librosa_compatibility_cpu_test.py'
            ' --ignore=test/torchaudio_unittest/prototype/functional/librosa_compatibility_cuda_test.py'
            ' --ignore=test/torchaudio_unittest/prototype/transforms/librosa_compatibility_cpu_test.py'
            ' --ignore=test/torchaudio_unittest/prototype/transforms/librosa_compatibility_cuda_test.py'
            ' --ignore=test/torchaudio_unittest/prototype/transforms/librosa_compatibility_cpu_test.py'
            ' --ignore=test/torchaudio_unittest/prototype/transforms/librosa_compatibility_cuda_test.py'
            ' --ignore=test/torchaudio_unittest/prototype/hifi_gan/hifi_gan_cpu_test.py'
            ' --ignore=test/torchaudio_unittest/prototype/hifi_gan/hifi_gan_gpu_test.py'
            ' test/torchaudio_unittest/'
        ),
```
It's either this or pulling in librosa as a build dependency (which would be a better solution, but then someone has to create that EC first)